e2e/tests/api-product-list.spec.ts

```
@@ -633,35 +581,15 @@ test.describe('APIProduct List Page - Combined Filters', () => {
-
-    // Apply namespace filter
-    const searchInput = page.locator('input[aria-label="Resource search"]');
-    await searchInput.fill('kuadrant-test');
+    await fillTextFilter(page, 'Namespace', 'kuadrant-test');
     await page.waitForTimeout(1000);
 
     // Verify both filter labels are shown
-    await expect(page.locator('.pf-m-outline:has-text("Published")')).toBeVisible();
-    await expect(page.locator('.pf-m-outline:has-text("kuadrant-test")')).toBeVisible();
+    await expect(page.locator('.pf-v6-c-label.pf-m-filled:has-text("Published")')).toBeVisible();
+    await expect(page.locator('.pf-v6-c-label.pf-m-filled:has-text("kuadrant-test")')).toBeVisible();
```

ESLint

[error] Replace `page.locator('.pf-v6-c-label.pf-m-filled:has-text("kuadrant-test")')` with `⏎······page.locator('.pf-v6-c-label.pf-m-filled:has-text("kuadrant-test")'),⏎····`

(prettier/prettier)